src/auto/agents/manager.rs

```diff
@@ -40,28 +56,47 @@ r#"
 Personality: {personality}
 
 You will be given one task.
-Your goal is to split that task up into smaller tasks.
-Your goal is to delegate those subtasks, one at a time.
+Your goal is to understand the task, split it into smaller tasks when necessary.
```

Comment:
tell the agent to stay within capabilities scope
ask the agent to go straight/quickly to commands